usecases/connect-autonomous-database/src/main/java/com/oracle/cloud/world/demo/MainApplication.java

```diff
@@ -0,0 +1,12 @@
+package com.oracle.cloud.world.demo;
```

Comment:
Should this be generalized (not referencing Cloud World)?

Reply:
Renamed the package name